### ISSUE TITLE
Startup Popup has the OpenToonz logo

### ISSUE THREAD

**Issue by turtletooth** [motionmonster/motionmonster#52]
I need a new MotionMonster small logo.  The current one is horrible and I am waiting until I make a new one to update the graphic.

**Comment by RodneyBaker** [comment]
I *almost* commented on adding an image to the startup over at OpenToonz Central but anticipated one was coming.  

I've been drawing (mostly cute) monsters since you launched Motion Monster but none quite look the part. :)

**Comment by RodneyBaker** [comment]
Not suggesting this for a logo but thought you might find it interesting.
I started sketching to see if I could come up with a logo but wasn't getting anywhere so I decided to experiment in Motion Monster (previous release pending install of the latest and greatest) with emphasis on the Type Tool and adding FX in Schematic view .  

This was the result.
The hand/claw could use some better movement but I like the general effect and colors.

![motionmonstersmlogo](https://cloud.githubusercontent.com/assets/2576432/18346091/2f5344a2-7584-11e6-83d5-9430725ff18b.gif)
